From c14854e7d66a45742342a6e217af1c7901eeb92d Mon Sep 17 00:00:00 2001
From: Carlos Maiolino <cem@kernel.org>
Date: Fri, 19 Jul 2024 12:57:36 +0200
Subject: [PATCH] metadata_csum_seed and orphan_file features should be off by default

Resolves: RHEL-49800

These settings were enabled by default upstream. We don't support them
in RHEL and we don't want to, so disable them by default on mkfs.
Because this configuration is default only for rhel, this is of course a
RHEL-Only patch.

Signed-off-by: Carlos Maiolino <cmaiolino@redhat.com>
---
 misc/mke2fs.conf.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/mke2fs.conf.in b/misc/mke2fs.conf.in
index b7fc95df..05680992 100644
--- a/misc/mke2fs.conf.in
+++ b/misc/mke2fs.conf.in
@@ -11,7 +11,7 @@
 		features = has_journal
 	}
 	ext4 = {
-		features = has_journal,extent,huge_file,flex_bg,metadata_csum,metadata_csum_seed,64bit,dir_nlink,extra_isize,orphan_file
+		features = has_journal,extent,huge_file,flex_bg,metadata_csum,64bit,dir_nlink,extra_isize
 	}
 	small = {
 		blocksize = 1024
-- 
2.45.2

